intbot/core/tasks.py

```diff
@@ -66,14 +67,35 @@ def process_github_webhook(wh: Webhook):
         channel_id=channel.channel_id,
         channel_name=channel.channel_name,
         content=f"GitHub: {parsed.as_discord_message()}",
-        # Mark as unsend - to be sent with the next batch
+        # Mark as unsent - to be sent with the next batch
         sent_at=None,
     )
     wh.processed_at = timezone.now()
     wh.save()
 
 
 def process_zammad_webhook(wh: Webhook):
-    # NOTE(artcz) Do nothing for now. Just a placeholder.
-    # Processing will come in the next PR.
-    return
+    if wh.source != "zammad":
+        raise ValueError("Incorrect wh.source = {wh.source}")
+
+    # Unlike in github, the zammad webhook is richer and
+    # contains much more information, so no extra fetch is needed.
+    # However, we can extract information and store it in the meta field, that
+    # way we can reuse it later more easily.
+    wh = prep_zammad_webhook(wh)
+    channel = discord_channel_router(wh)
+
+    if channel == dont_send_it:
+        wh.processed_at = timezone.now()
+        wh.save()
```

Comment:
Why is it saved if it's an empty channel? I don't understand the motivation behind. I'd maybe add a comment, because this is not that clear and intuitive to me.

Reply:
It's marking it as processed so next time we want to reprocess all the not processed webhooks this one will be skipped.